app/server.py

```diff
@@ -48,7 +48,7 @@ def search(search_engine):
             err = [400, 'Not Found - missing query', qformat]
             return bad_request(err)
 
-        result = feedgen(query, engine[0], count)
+        result = feedgen(query, engine, count)
```

Comment:
This change needs to be matched by a similar change in https://github.com/fossasia/query-server/blob/master/app/scrapers/__init__.py#L11  If `server.py` is going to send the full name of the engine (a good idea) then `scrapers/__init__.py` needs to be set up to receive the full name instead of just the first letter.